modes/dired/evil-collection-dired.el

```diff
@@ -186,7 +186,17 @@
     ";d" 'epa-dired-do-decrypt
     ";v" 'epa-dired-do-verify
     ";s" 'epa-dired-do-sign
-    ";e" 'epa-dired-do-encrypt))
+    ";e" 'epa-dired-do-encrypt)
+
+    ;; dired-subtree commands
+    (with-eval-after-load 'dired-subtree
+	(evil-collection-define-key 'normal 'dired-mode-map
+	    "I" 'dired-subtree-toggle
+	    (kbd "TAB") 'dired-subtree-cycle
+	    "gj" 'dired-subtree-up
```

Comment:
Doesn't this go up so should be gk.

Reply:
This is a plain typo, sorry. It should be `gh` as I wanted it to behave similar to `org-mode` where `gh` brings me one indentation level up and conversely `gl` brings me one indentation level down.

Reply:
Sounds good, gh/gl OK with me.

BTW, thanks for reminding me of dired-sidebar. I finally summoned the effort to bring it to evil-collection.

Reply:
I just saw that you are the author of `dired-sidebar`, made me wonder why it wasn't part of `evil-collection` before. :) Nice little package, I really like that it carries over any settings from plain `dired`.

Reply:
This should be gh right?

Reply:
Yes, fixed with --amend and force push now.